langgraph_supervisor/supervisor.py

```diff
@@ -17,14 +18,16 @@
 from langgraph_supervisor.agent_name import AgentNameMode, with_agent_name
 from langgraph_supervisor.handoff import (
     METADATA_KEY_HANDOFF_DESTINATION,
+    METADATA_KEY_IS_HANDOFF_BACK,
     create_handoff_back_messages,
     create_handoff_tool,
 )
 
-OutputMode = Literal["full_history", "last_message"]
+OutputMode = Literal["full_history", "only_history", "last_message"]
 """Mode for adding agent outputs to the message history in the multi-agent workflow
 
 - `full_history`: add the entire agent message history
+- `only_history`: add only the supervisor's message history, without the AI message or handoff messages
```

Comment:
this is not exactly what it's doing, right? it's including internal agent history as well, not just supervisor's messages

could we maybe do `"no_handoffs"` or `"full_history_no_handoffs"` instead? `only_history` is a bit confusing

Reply:
(was incorrect anyway - ported the wrong thing)